tests/detection/test_map.py

```diff
@@ -242,6 +334,21 @@ def test_map(self, compute_on_cpu, ddp):
             metric_args={"class_metrics": True, "compute_on_cpu": compute_on_cpu},
         )
 
+    @pytest.mark.parametrize("ddp", [False])
```

Comment:
Do it work in ddp mode?
```suggestion
    @pytest.mark.parametrize("ddp", [False, True])
```